recipes/libad9361-iio/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+libudev
```

Comment:
Please use the libudev package in conda-forge

Reply:
I'm using the `libudev-devel` CDT, and I thought having this in `yum_requirements.txt` was necessary so that the library is available during the testing phase? I can try it again without it, but I vaguely remember running into a missing library error in the test phase.

Reply:
Ahh ok